xmtp_id/Cargo.toml

```diff
@@ -22,9 +24,11 @@ rand.workspace = true
 serde.workspace = true
 sha2 = "0.10.8"
 thiserror.workspace = true
+tokio.workspace = true
 tracing.workspace = true
 xmtp_cryptography.workspace = true
 xmtp_proto = { workspace = true, features = ["proto_full"] }
+xmtp_mls.workspace = true
```

Comment:
Anything you need from `xmtp_mls` should get copied in to this crate, since we don't want a circular dependency.